spec/controllers/v3/repositories_controller_spec.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe V3::RepositoriesController do
+  subject { create :repository_compound }
+  let!(:user) { subject.owner }
+
+  context 'successful' do
+    describe 'action: index' do
+      before do
+        get :index, params: {organizational_unit_id: user.to_param}
+      end
+
+      it { expect(response).to have_http_status(:ok) }
```

Comment:
Please add another `it` that checks if `repository` and `organizationalUnit` is not nil.
```ruby
it 'finds the organizational unit' do
  expect(response_data['organizationalUnit']).not_to be(nil)
end
it 'finds the repository' do
  expect(response_data['organizationalUnit']['repository']).not_to be(nil)
end
```